.github/workflows/push-release.yml

```diff
@@ -504,21 +507,22 @@ jobs:
           mkdir -p $HOME/bin
           touch $HOME/bin/rancher-rke-k8s-versions.txt
       - name: Create components and images files
-        run: sudo -E ./scripts/create-components-images-files.sh
+        shell: bash
+        run: ./scripts/create-components-images-files.sh
       - name: Move files
         run: |
           mv $HOME/bin/* ./dist
           mv ./bin/*.txt ./dist
           mv ./bin/*.sh ./dist
           mv ./bin/*.ps1 ./dist
       - name: Create sha256sum.txt file
-        run: ./scripts/artifacts-hashes.sh
+        run: bash ./scripts/artifacts-hashes.sh
       - name: Upload artifacts
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: gh release upload -R ${{ env.REPOSITORY_OWNER }}/rancher ${{ env.TAG }} ./dist/* --clobber
   docker-image-digests:
-    if: github.event_name == 'tag' && contains(github.ref, 'tags/') # Only run on tag
+    if: github.event_name == 'push' && contains(github.ref, 'tags/') # Only run on tag
```

Comment:
Question, is the comment and the `contains()` checking against `tags` right?

Reply:
Yes, when a tag is created the `event_name` isn't `tag`, it actually is `push`, but the ref contains `tags/`